src/app/models/Job.ts

```diff
@@ -2,9 +2,9 @@ export interface Job {
   id: string;
   name: string;
   image: string;
-  creation_time: Date;
-  start_time: Date;
-  end_time: Date;
+  creation_time: string;
```

Comment:
How do these times come over the wire? Are they indeed strings or can they be stored as Date objects?

Reply:
Reverting these, I was debugging some model bugs I had but didn't mean to keep them as strings